functests/2_performance_update/updating_profile.go

```diff
@@ -195,18 +196,35 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 		table.DescribeTable("Verify that profile parameters were updated", func(cmd, parameter []string, shouldContain bool, useRegex bool) {
 			for _, node := range workerRTNodes {
 				for _, param := range parameter {
-					result, err := nodes.ExecCommandOnNode(cmd, &node)
-					Expect(err).ToNot(HaveOccurred(), "failed to execute %s", cmd)
-
-					matcher := ContainSubstring(param)
-					if useRegex {
-						matcher = MatchRegexp(param)
-					}
-
-					if shouldContain {
-						Expect(result).To(matcher)
+					if strings.Contains(cmd[1], "nr_hugepages") {
+						numaInfo, err := nodes.GetNumaNodes(&node)
+						Expect(err).ToNot(HaveOccurred(), "failed to get numa nodes")
+						count := 0
+						for i:=0; i < len(numaInfo); i++ {
+							n := strconv.Itoa(i)
+							str := "node" + n 
+							numaNodeHugePage := strings.Replace(cmd[1], "nodeN", str, 1)
+							cmd = []string{"cat", numaNodeHugePage}
+							result, err := nodes.ExecCommandOnNode(cmd, &node)
+							Expect(err).ToNot(HaveOccurred(), "failed to execute %s", cmd)
+							t,_ := strconv.Atoi(result)
```

Comment:
it bad practice to ignore the error
please use `Expect(err).ToNot(HaveOccurred())`

Reply:
okay, updating the patch.